src/ProjectionUtil/ProjectionUtil.js

```diff
@@ -98,6 +98,49 @@ export class ProjectionUtil {
       }
     }
   }
+
+  /**
+   * Converts geographic coordinates given in DDD format like `DD.DDDD°` to
+   * the degree, minutes, decimal seconds (DMS) format like
+   * `DDD° MM' SS.SSS"`.
+   *
+   * @param {number} value Value to be converted.
+   *
+   * @return {string} Converted value.
+   */
+  static dToDms(value) {
+    const deg = parseInt(value, 10);
+    const min = parseInt((value - deg) * 60, 10);
+    const sec = ((value - deg - min / 60) * 3600);
+    return `${deg}° ${ProjectionUtil.zerofill(min)}' ${ProjectionUtil.zerofill(sec.toFixed(2))}''`;
```

Comment:
Shouldn't this end with `"` instead of two single `'`?

Reply:
It's fine, see [here](https://en.wikipedia.org/wiki/Degree_(angle)#Subdivisions).